frontend/scss/app/components/source_editor/_source_editor_form.scss

```
@@ -21,7 +21,8 @@ $form-width: 480px;
 
 .tp__source-editor-form {
 
-  padding: 0 var(--padding-micro);
+  padding: 0 var(--padding-mini) var(--padding-mini) var(--padding-mini);
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)